### DISCUSSION TITLE
Button to translate the community web page

### DISCUSSION THREAD

**Discussion by StevieSenpai** [gameserverapp/Platform#575] [Suggestions]
it would be nice to add the same button that is for the administrative part for the community website so that each user can choose their language and the page will be displayed in the selected language.
![imagen_2023-11-24_202445602](https://github.com/gameserverapp/Platform/assets/150964621/8791c15d-acff-4e7f-8b77-dd40ab3f0df0)

**Comment by zerafox** [comment]
This would be nice as we have a few non-english speaking or overly fluent users.
Being able to translate the Community Website would be a huge help.
